scrapyrt/core.py

```diff
@@ -39,6 +39,8 @@ def crawl(self, *args, **kwargs):
         self.crawling = True
         try:
             self.spider = self._create_spider(*args, **kwargs)
+            self._apply_settings()
+            self._update_root_log_handler()
```

Comment:
In Scrapy we try to go case by case when it comes to maintaining backward compatibility, instead of having a general rule. Here specifically it seems like keeping backward compatibility would not be a maintenance pain. I think if this were Scrapy we would be keeping backward compatibility.

But this is indeed not Scrapy, and I am OK with the approach you decided on. I just though it was worth pointing it out.